README.rst

```diff
@@ -157,3 +157,70 @@ contains metadata about a docker repository.
 
 These files are produced by a publish action in
 `Pulp <http://www.pulpproject.org>`_.
+
+Crane Admin
+-----------
+
+List of images served by Crane can be obtained by opening ``/crane/repositories`` in web browser or
+with ``curl``. The default Apache configuration distributed with Crane restricts access to this URL
+from ``localhost`` only. When accessed from a web browser, a page listing all repositories and
+their images is returned. This URL accepts a "Content-Type" header. When "application/json" is
+specified, the application responds with JSON that has the following format:
+
+.. code-block::
+
+    {
+        "pulpdemo-busybox": {
+            "image_ids": [
+                "2982ec56c8d910121e7594ca7890b062f6d37fadf7575f6a6f3adbabbafac9f5",
+                "2aed48a4e41d3931167146e9b7492aa5639e7f6478be9eac584726ecec6824ed",
+                "492dad4279bae5bb73648efe9bf467b2cfa8bab1d593595226e3e7a95d9f6c35",
```

Comment:
Maybe trim some of the redundant lines from the image_ids sections of the example? It's up to you, I'm ok with it as is.